### PR TITLE
Update README.md and docs root to clarify versioning

### LINKED ISSUE #618: Update ReadMe to reflect quality levels of client libraries
We will soon have client libraries at different quality levels. Let's make sure we reflect this information in our top-level ReadMe file the moment we cut the new release.

The update for [PHP's ReadMe](https://github.com/omaray/google-cloud-php/blob/readme-update-beta/README.md) has already be done (as a [Pull Request](https://github.com/GoogleCloudPlatform/google-cloud-php/pull/274) for now) and can be used as the canonical example to follow. The list below highlights the main changes to follow for this repo.

## General
- [ ] P1: change from `Google Cloud Libraries for .NET` to `Google Cloud .NET Client` (for consistency)

## Listing Beta client libraries
- [ ] Remove the text "The Google Cloud Libraries for .NET support the following Google Cloud Platform services: "

- [ ] Create a section to show the Beta client libraries (see image right below for text & layout)

- [ ] Make the term `Beta` link to the **Versioning** section of the ReadMe (see image right below)

- [ ] Add the term `(Beta)` next to each API name (see image right below for text and visual):

image

## Listing Alpha client libraries
- [ ] Create a section to show the Alpha client libraries (see image right below for text and layout)

- [ ] Make the term `Alpha` link to the **Versioning** section of the ReadMe (see image right below)

- [ ] Add the term `(Alpha)` next to each API name (see image right below for text and visual):

image

## Versioning section

- [ ] If the **Versioning** section doesn't exist, add one.

- [ ] Add the following content in the **Versioning** section:

image

### PR DESCRIPTION
**Pull request by jskeet**
Fixes #618.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,11 +88,27 @@ See [CONTRIBUTING.md](./CONTRIBUTING.md) for more information on how to get star
...
-considered stable - anything may change at any time.
-
-Anything with a suffix after the three numbers (such as `1.0.0-beta01`) is expected to work,
-but further API changes may occur before the next stable release.
+The version number in the package *name* (and .NET namespace)
+represents the version of the underlying API; the version number for
+the package itself indicates the status of the client library, according to
+the following quality levels:
+
+**Late Beta**: Libraries defined at a Beta quality level are
```
Comment (evildour) [inline README.md]: "Beta" => "Late Beta"

File: README.md [hunk 2]
```diff
@@ -8,21 +8,26 @@
...
+* [Google BigQuery](https://cloud.google.com/bigquery/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.BigQuery.V2/) (late beta)
+* [Google Cloud Datastore](https://cloud.google.com/datastore/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Datastore.V1/) (late beta)
+* [Google Stackdriver Logging](https://cloud.google.com/logging/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Logging.V2/) (late beta)
+
+The following libraries are available at a [beta](#versioning) quality level:
+
+* [Google Cloud Pub/Sub](https://cloud.google.com/pubsub/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.PubSub.V1/) (beta)
+* [Google Cloud Natural Language](https://cloud.google.com/natural-language/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Language.V1Beta1/) (beta)
+
+The following libraries are available at a [alpha](#versioning) quality level:
```
Comment (evildour) [inline README.md]: "a" => "an"

Comment (jskeet) [inline README.md]: Fixed, thanks, along with the beta => late beta.

**Comment by evildour** [comment]
LGTM

**Comment by omaray** [comment]
LGTM

**Pull request merged**